package.json

```diff
@@ -2,7 +2,7 @@
   "name": "data-transparency-ui",
   "version": "1.0.0",
   "description": "Library of UI components powering Data Transparency websites",
-  "main": "dist/index.js",
```

Comment:
@MaxwellKendall

Reply:
![image](https://user-images.githubusercontent.com/12897813/101175705-7cbd6e80-3613-11eb-9bdd-4cdbd609ab78.png)

See third todo on PR checklist for author.